apps/navstar_dns/src/navstar_dns_listener.erl

```diff
@@ -12,7 +12,7 @@
 -behaviour(gen_server).
 
 %% API
--export([start_link/0, convert_zone/3]).
+-export([start_link/0, convert_zone/3, retry_spartan/1]).
```

Comment:
I think you might only need to export it if you're calling it with the `module:fun()` format? Although I'd have to check.